resources/files/razor-server.sh

```diff
@@ -0,0 +1,5 @@
+# Add /opt/puppetlabs/bin to the path for sh compatible users
```

Comment:
You had asked if we care about C shells, and the answer is "I don't know, but we can probably replicate this for C shells too if you want to"